builders/xcm/xcm-transactor.md

```diff
@@ -0,0 +1,194 @@
+
+Once the transaction is processed, you can check the relevant extrinsics and events in [Moonbase Alpha](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbase.moonbeam.network#/explorer/query/0x14cebfb2b4a4c0bf72cb2562344e6803263f45491d2ab14e7b91115ebd52e706){target=_blank} and the [relay chain](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffrag-moonbase-relay-rpc-ws.g.moonbase.moonbeam.network#/explorer/query/0x630456e6578d5b414db2e83486a6961ff90dc06bf979988b370dcb0e57550e41){target=_blank}. Note that, in Moonbase Alpha, there is an event associated to the `transactThroughDerivative` method, but also some `xcUNIT` tokens are burned to repay the sovereign account for the transactions fees. In the relay chain, the `paraInherent.enter` extrinsic shows a `balance.Transfer` event, where 1 `UNIT` token is transferred to Alice's address. Still, the transaction fees are paid by Moonbase Alpha sovereign account.
+
+## Retrieve Registered Derivative Indexes
+
+To fetch a list of all registered addresses allowed to operate through the Moonbeam-based network sovereign account and their corresponding indexes, head to the **Chain State** section of [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbase.moonbeam.network#/chainstate) (under the **Developer** tab). In there, take the following steps:
+
+1. From the **selected state query** dropdown, choose **xcmTransactor**
+2. Select the **indexToAccount** method
+3. (Optional) Disable/enable the include options slider. This will allow you to query the address authorized for a given index or request all addresses for all registered indexes
+4. If you've enabled the slider, enter the index to query
+5. Send the query by clicking on the **+** button
+
+![Check Registered Derivative Indexes](/images/builders/xcm/xcm-transactor/xcmtransactor-2.png)
+
+## XCM-Transactor Precompile {: #xcmtransactor-precompile}
+
+The XCM-Transactor precompile contract allows developers to access the XCM-Transactor pallet features through the Ethereum API of Moonbeam-based networks. Similar to other [precompile contracts](/builders/build/canonical-contracts/precompiles/){target=_blank}, the XCM-Transactor precompile is located at the following addresses:
+
+=== "Moonbeam"
+     ```
+     {{networks.moonbeam.precompiles.xcm_transactor}}
+     ```
+
+=== "Moonriver"
+     ```
+     {{networks.moonriver.precompiles.xcm_transactor}}
+     ```
+
+=== "Moonbase Alpha"
+     ```
+     {{networks.moonbase.precompiles.xcm_transactor}}
+     ```
+
+### The XCM-Transactor Solidity Interface {: #xcmtrasactor-solidity-interface } 
+
+[XcmTransactor.sol](https://github.com/PureStake/moonbeam/blob/master/precompiles/xcm-transactor/XcmTransactor.sol){target=_blank} is an interface through which developers can interact with the XCM-Transactor pallet using the Ethereum API.
+
+The interface includes the following functions:
```

Comment:
Were the signed variants of the transact functions omitted purposefully?

Reply:
Yes, this is still being actively developed so there will be new functions that will need to be added/modified in the future